tests/test-squid.py

```diff
@@ -1,10 +1,14 @@
 import boto3
```

Comment:
Create a new file `tests/test-csv.py` and add your code to that instead of putting in `tests/test-squid.py`